django/db/migrations/executor.py

```diff
@@ -82,9 +82,15 @@ def migrate(self, targets, plan=None, fake=False, fake_initial=False):
         all_backwards = all(backwards for mig, backwards in plan)
 
         if not plan:
-            # Nothing to do for an empty plan, except for building the post
-            # migrate project state
+            # The resulting state should still include applied migrations.
```

Comment:
Not sure what "still" is referring to in this comment and the similar comment in the next commit. "still" might not be needed.